src/polygon.typ

```diff
@@ -57,3 +59,48 @@
     none
   }
 }
+
+// Calculate triangle centroid
+#let triangle-centroid(points) = {
+  assert.eq(points.len(), 3)
+
+  let (mx, my, mz) = (0, 0, 0)
+  for p in points {
+    let (x, y, z) = p
+    mx += x
+    my += y
+    mz += z
+  }
+  return (mx / 3, my / 3, mz / 3)
+}
```

Comment:
This function feels redundent if we have a general method anyway

Reply:
Gives correct points for 3D triangles, the other might not.